sentry/src/main/java/io/sentry/logger/LoggerApi.java

```diff
@@ -146,8 +167,20 @@ private void captureLog(
   }
 
   private @NotNull HashMap<String, SentryLogEventAttributeValue> createAttributes(
-      final @NotNull String message, final @NotNull SpanId spanId, final @Nullable Object... args) {
+      final @Nullable Map<String, Object> incomingAttributes,
+      final @NotNull String message,
+      final @NotNull SpanId spanId,
+      final @Nullable Object... args) {
     final @NotNull HashMap<String, SentryLogEventAttributeValue> attributes = new HashMap<>();
+
+    if (incomingAttributes != null) {
```

Comment:
Should we create a copy of the attributes (here or in the capture method) to avoid having a map that could be changed while we iterate on it?